deploy/helm-charts/templates/secret.yaml

```diff
@@ -1,11 +1,22 @@
 apiVersion: v1
 kind: Secret
+type: Opaque
 metadata:
   name: {{ .Values.secret_name }}
   namespace: {{ .Release.Namespace }}
-type: Opaque
 data:
-  key: |-
-    {{ .Files.Get "data/server.key" | b64enc }}
-  cert: |-
-    {{ .Files.Get "data/server.crt" | b64enc }}
+{{- if and (eq "" .Values.secrets.tlsCertFilePath) (eq "" .Values.secrets.tlsKeyFilePath) }}
```

Comment:
either we should use `or` operator here or we should have additional else if cases that handle the cases where key is blank but cert is defined and vice versa.

Reply:
`and` would do the trick in this case, if any of the 2 keys are missing, it will error out somewhat like this: `unknown object type "nil" in Secret.data.tls.crt`

Reply:
great!